torch/_inductor/codegen/common.py

```diff
@@ -805,8 +819,21 @@ class CSEProxy:
             @staticmethod
             def __getattr__(name):
                 def inner(*args, **kwargs):
+                    # TritonTemplateKernel has no current_node
+                    buf_bounds = ValueRanges.unknown()
+                    if hasattr(V.interpreter, "current_node"):
+                        fx_node = V.interpreter.current_node
+                        buf_bounds = self.bounds.get(fx_node, ValueRanges.unknown())
```

Comment:
in some of the other places `self.bounds` is a `ValueRanges` object and in some places `self.bounds` is a dict.. maybe using `value_range` for the instance would be clearer. or type annotations.

Reply:
Changed name to `node_to_bounds` and added type annotations.